Misc/NEWS.d/next/Security/2024-08-06-00-06-23.gh-issue-112301.4k4lw6.rst

```diff
@@ -0,0 +1,2 @@
+Add ability to ignore warnings per file with warning count in warning checking tooling.
```

Comment:
This PR depends on already reviewed #122711 and https://github.com/python/devguide/pull/1364 to merge